.github/workflows/download_onesky_translations.yml

```diff
@@ -1,36 +1,44 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Download And Commit OneSky Translations
 
-# Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the develop branch
-  push:
-    branches: [ develop ]
-  pull_request:
-    branches: [ develop ]
+  schedule:
+    - cron: "0 23 * * *"
```

Comment:
there is also a trigger that allows you to manually trigger it from the GHA interface. We would probably want to include that with this workflow